omod/src/test/java/org/openmrs/module/fhir2/providers/BaseFhirResourceProviderWebTest.java

```diff
@@ -104,6 +107,7 @@ public String getGlobalProperty(String property) throws APIException {
 		});
 		
 		servlet.setResourceProviders(getResourceProvider());
+		servlet.setMessageSource(messageSource);
```

Comment:
Setting a `ReloadableResourceBundleMessageSource` as `messageSource` for web tests.